src/Components/About/ListMemberNGO/ListMemberNGO.js

```diff
@@ -0,0 +1,14 @@
+import Member from "./Member";
+import "./Member.css";
+
+const ListMemberNGO = ({ members }) => {
+  return (
+    <div className="list-member-container">
+      {members.map((member) => {
+        return <Member member={member} key={member.name} />;
+      })}
```

Comment:
Acá te podes evitar el return haciendo lo siguiente 😄 

```suggestion
      {members.map((member) => <Member member={member} key={member.name} />)}
```